model/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala

```diff
@@ -463,4 +463,75 @@ class WorkspaceModelSpec extends AnyFreeSpec with Matchers {
       Attributable.attributeCount(map6) shouldBe 6
     }
   }
+
+  "Safe printer" - {
+    "should safely print attributes" in {
+      val map1 = Map(
+        AttributeName("namespace1", "name1") -> AttributeString("value1")
+      )
+      val map2: AttributeMap = Map.empty
+      val map3 = Map(
+        AttributeName("namespace1", "name1") -> AttributeValueEmptyList
+      )
+      val map4 = Map(
+        AttributeName("namespace1", "name1") -> AttributeValueList(
+          Seq(
+            AttributeNumber(1), AttributeNumber(2), AttributeNumber(3)
+          )
+        )
+      )
+      val map5 = Map(
+        AttributeName("namespace1", "name1") -> AttributeString("value1"),
+        AttributeName("namespace2", "name2") -> AttributeString("value2")
+      )
+      val map6 = Map(
+        AttributeName("default", "read_counts") ->
+          AttributeValueList(
+            Vector(
+              AttributeString("gs://my-workflow/shard-0/cacheCopy/some_file.tsv"),
+              AttributeString("gs://my-workflow/shard-1/cacheCopy/some_file.tsv"),
+              AttributeString("gs://my-workflow/shard-2/cacheCopy/some_file.tsv"),
+              AttributeString("gs://my-workflow/shard-3/cacheCopy/some_file.tsv"),
+              AttributeString("gs://my-workflow/shard-4/cacheCopy/some_file.tsv"),
+              AttributeString("gs://my-workflow/shard-5/cacheCopy/some_file.tsv")
+            )
+          )
+        )
+
+      val map7 = Map(
+        AttributeName("default", "read_counts") -> AttributeEntityReference("type", "name"))
+
+      val map8 = Map(
+        AttributeName("default", "read_counts") -> AttributeEntityReferenceList(Seq(AttributeEntityReference("type", "name1"), AttributeEntityReference("type", "name2")))
+      )
+
+      // We forbid nested lists. `AttributeValueList` requires its contents to be an `AttributeValue` and not another `AttributeValueList`
+      // Does not compile:
+      // val map7 = Map(
+      //   AttributeName("default", "read_counts") ->
+      //     AttributeValueList(
+      //       Vector(
+      //         AttributeValueList(
+      //           Vector(
+      //             AttributeString("gs://my-workflow/shard-0/cacheCopy/some_file.tsv"),
+      //             AttributeString("gs://my-workflow/shard-1/cacheCopy/some_file.tsv"),
+      //             AttributeString("gs://my-workflow/shard-2/cacheCopy/some_file.tsv"),
+      //             AttributeString("gs://my-workflow/shard-3/cacheCopy/some_file.tsv"),
+      //             AttributeString("gs://my-workflow/shard-4/cacheCopy/some_file.tsv"),
+      //             AttributeString("gs://my-workflow/shard-5/cacheCopy/some_file.tsv")
+      //         )
+      //       )
+      //     )
+      // )
```

Comment:
Neither, I intended it as a permanent comment - hence the indented start to the `//`

Do you disapprove?

Reply:
Ah, ok, consider this a TOL level comment then. 

Maybe just changing `// Does not compile:` at the start to `// For example, this would not compile:` would help my quick-scan brain have an easier time telling that it's here deliberately?

Reply:
Agreed & implemented!